pom.xml

```diff
@@ -575,6 +600,12 @@
       </dependency>
 
       <dependency>
+        <groupId>com.google.errorprone</groupId>
```

Comment:
Where is this dependency being used?

Reply:
Guava brings it in, and once it's in Immutables adds it to the generated code, if we don't declare it we get:

```
[INFO] --- maven-dependency-plugin:3.0.1:analyze-only (default) @ rest ---
[WARNING] Used undeclared dependencies found:
[WARNING]    com.google.errorprone:error_prone_annotations:jar:2.0.18:compile
```